src/components/Recorder/recorder.css

```diff
@@ -0,0 +1,10 @@
+.container {
+  margin: 0 auto;
+  padding: 10px 0;
+  max-width: 950px;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+}
```

Comment:
[Can we get this setup please](https://www.npmjs.com/package/eslint-plugin-css-modules). 

I basically rank un-alphabetical css up there as an annoyance of equal importance to world hunger.